shredstream.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package shredstream;
+
+import "google/protobuf/empty.proto";
+import "shared.proto";
+
+message Heartbeat {
+  // don't trust IP:PORT from tcp header since it can be tampered over the wire
```

Comment:
Can you add a comment that the socket's IP must equal the IP it's coming from? Ports can be diff